distribution/docs/src/main/resources/content/_managing/_configuring/hardening-solr.adoc

```diff
@@ -82,17 +82,17 @@ ${branding} distribution.
 
 . Edit the `${home_directory}/solr/server/solr/security.json` file. Solr stores a salted hash of
 the user passwords in this file.
-. Assuming the Solr username is `ddf`, change the credentials section to match
+. Assuming the Solr username is `admin`, change the credentials section to match
 this string:
 +
 ```
  "credentials": {
-      "ddf": "31u3dZLhFmlhsyYzlLxtUDoKR2j8oRnLwWcAH7Lbor4= F9eRJs+ZMI95YSQnBP/lGrLH8h60lKKizjMgC69J+WM="}
+      "admin": "31u3dZLhFmlhsyYzlLxtUDoKR2j8oRnLwWcAH7Lbor4= F9eRJs+ZMI95YSQnBP/lGrLH8h60lKKizjMgC69J+WM="}
```

Comment:
This password hash will be different. It won't be `31u3dZLhFmlhsyYzlLxtUDoKR2j8oRnLwWcAH7Lbor4= F9eRJs+ZMI95YSQnBP/lGrLH8h60lKKizjMgC69J+WM=`.

It will be: `EjjOS/zyQ1KQQdSXFb/rFm7w6MItU5pmdthM35ZiJaA= ZZI7d4jf/8hz5oZz7ljBE6+uv1wqncj+VudX3arbib4=` for password `admin`